tests/run_tests.sh

```diff
@@ -220,13 +220,13 @@ while getopts "h?c:d:e:f:i:k:l:m:n:op:rs:t:ux" opt; do
             show_help_and_exit 0
             ;;
         c )
-            TEST_CASES=${OPTARG}
+            TEST_CASES="${TEST_CASES} ${OPTARG}"
             ;;
         d )
             DUT_NAME=${OPTARG}
             ;;
         e )
-            EXTRA_PARAMETERS=${OPTARG}
+            EXTRA_PARAMETERS="${EXTRA_PARAMETERS} ${OPTARG}"
```

Comment:
How is this consumed in the testcase? How do we differentiate between single arg vs key value pair?

Reply:
pytest would handle the difference, the script would just pass the parameter list to pytest.